client/components/seo/seo-site-search-preview.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+
+export const SeoSiteSearchPreview = React.createClass( {
+	shouldComponentUpdate( next ) {
+		const current = this.props;
+
+		return ! (
+			current.snippet === next.snippet &&
+			current.title === next.title &&
+			current.url === next.url
+		);
+	},
+
+	render() {
+		const {
+			snippet,
+			title,
+			url
+		} = this.props;
+
+		return (
+			<div className="seo-search-result-preview">
```

Comment:
@aduth and @mtias - my thought was that this directory would also contain an OpenGraph Facebook preview and also possibly a blog-post-seo-results-preview which I was going to give another name. They are all pretty seo-specific and so I wanted to group them in the same directory. 

Do you think I should split them into separate directories?

How do you suggest handling the two-tiered CSS class names? `see__search-result-preview__title` ?